about/index.css

```diff
@@ -220,19 +220,67 @@ li {
 }
 
 @media only screen and (min-width: 769px) {
```

Comment:
- [ ] (OPTIONAL) Good job using media queries to make your page responsive but your page breaks at a certain width and some features do not display well until certain widths. Kindly arrange your mobile view right up to `992px` and make sure your speakers' section displays the same in all desktop views.
- [ ] (OPTIONAL) Kindly fix the break in the page so as to improve your user interface. Below is a video of how your page functions right now.
- Your site:
![Animation](https://user-images.githubusercontent.com/85113567/159149709-88c176e5-0f2c-4d60-a1ed-37217649300c.gif)